packages/fxa-auth-server/test/local/payments/fixtures/customer1.json

```diff
@@ -53,6 +54,7 @@
         "current_period_end": 1568573770,
         "cancel_at_period_end": false,
         "ended_at": null,
+        "latest_invoice": "628031D-0002",
```

Comment:
Hmm from the [Stripe docs](https://stripe.com/docs/api/subscriptions/object#subscription_object-latest_invoice), it looks like `latest_invoice` only comes from a subscription object, not a customer object.

![image](https://user-images.githubusercontent.com/13018240/75400434-e1011400-58c4-11ea-8d7a-ea71dfcadbc9.png)

Reply:
I think github had collapsed, but this is in a subscription object, inside of the customer object